phnt/include/ntioapi.h

```diff
@@ -211,84 +211,85 @@ typedef struct _FILE_IO_COMPLETION_INFORMATION
     IO_STATUS_BLOCK IoStatusBlock;
 } FILE_IO_COMPLETION_INFORMATION, *PFILE_IO_COMPLETION_INFORMATION;
 
+// q - NtQueryInformationFile, s - NtSetInformationFile, d - NtQueryDirectoryFile[Ex]
 typedef enum _FILE_INFORMATION_CLASS
 {
-    FileDirectoryInformation = 1, // q: FILE_DIRECTORY_INFORMATION
-    FileFullDirectoryInformation, // q: FILE_FULL_DIR_INFORMATION
-    FileBothDirectoryInformation, // q: FILE_BOTH_DIR_INFORMATION
-    FileBasicInformation, // q; s: FILE_BASIC_INFORMATION; s (requires FILE_WRITE_ATTRIBUTES)
-    FileStandardInformation, // q: FILE_STANDARD_INFORMATION
+    FileDirectoryInformation = 1, // d: FILE_DIRECTORY_INFORMATION (requires FILE_LIST_DIRECTORY)
+    FileFullDirectoryInformation, // d: FILE_FULL_DIR_INFORMATION (requires FILE_LIST_DIRECTORY)
+    FileBothDirectoryInformation, // d: FILE_BOTH_DIR_INFORMATION (requires FILE_LIST_DIRECTORY)
```

Comment:
A bunch of these where changed from q to D? q=query, s=set there's no D.

Reply:
I added a comment a few lines above:
```c
// q - NtQueryInformationFile, s - NtSetInformationFile, d - NtQueryDirectoryFile[Ex]
```
The idea is to allow distinguishing directory-specific queries from everything else.

Reply:
If the infoclass is used with a different function then it's supposed to be included last (but before //since and // index):

```
q: FILE_DIRECTORY_INFORMATION (requires FILE_LIST_DIRECTORY) (NtQueryDirectoryFile[Ex])
s: FILE_DIRECTORY_INFORMATION (requires FILE_LIST_DIRECTORY) (NtQueryDirectoryFile[Ex]) // since WINBLUE
qs: FILE_DIRECTORY_INFORMATION (requires FILE_LIST_DIRECTORY) (NtQueryDirectoryFile[Ex]) // since WINBLUE // 100
```

Reply:
Okay, I'll update the PR 👍